client/landing/subscriptions/components/settings/site-settings/site-settings.tsx

```diff
@@ -79,28 +81,55 @@ const SiteSettings = ( {
 type SiteSettingsPopoverProps = SiteSettingsProps & {
 	onUnsubscribe: () => void;
 	unsubscribing: boolean;
+	onResubscribe: () => void;
+	resubscribing: boolean;
+	isDeleted: boolean;
 };
 
 export const SiteSettingsPopover = ( {
 	onUnsubscribe,
 	unsubscribing,
+	onResubscribe,
+	resubscribing,
+	isDeleted,
 	...props
 }: SiteSettingsPopoverProps ) => {
 	const translate = useTranslate();
 	return (
 		<SettingsPopover className="site-settings-popover">
-			<SiteSettings { ...props } />
+			{ ( close: () => void ) => (
+				<>
+					<SiteSettings { ...props } />
 
-			<hr className="subscriptions__separator" />
+					<hr className="subscriptions__separator" />
 
-			<Button
-				className={ classNames( 'unsubscribe-button', { 'is-loading': unsubscribing } ) }
-				disabled={ unsubscribing }
-				icon={ <UnsubscribeIcon className="settings-popover__item-icon" /> }
-				onClick={ onUnsubscribe }
-			>
-				{ translate( 'Unsubscribe' ) }
-			</Button>
+					{ isDeleted ? (
+						<Button
+							className={ classNames( 'resubscribe-button', { 'is-loading': resubscribing } ) }
+							disabled={ resubscribing }
+							icon={ <ResubscribeIcon className="settings-popover__item-icon" /> }
+							onClick={ () => {
+								onResubscribe();
+								close();
+							} }
+						>
+							{ translate( 'Resubscribe' ) }
+						</Button>
+					) : (
+						<Button
+							className={ classNames( 'unsubscribe-button', { 'is-loading': unsubscribing } ) }
+							disabled={ unsubscribing }
+							icon={ <UnsubscribeIcon className="settings-popover__item-icon" /> }
+							onClick={ () => {
+								onUnsubscribe();
+								close();
+							} }
+						>
+							{ translate( 'Unsubscribe' ) }
+						</Button>
+					) }
```

Comment:
Let's move it to a notification popup, and with that, these lines are no longer needed.